### ISSUE TITLE
Carthage build fails at building RxMoya (4.0.2)

### ISSUE THREAD

**Issue by aloco** [Moya/Moya#260]
Hi,

since the release of 4.0.2, ```Carthage``` fails at building ```RxMoya```

Sadly I am not very familiar with cocoapods, but I guess the issue is related with https://github.com/Moya/Moya/commit/f6a791398b47b4a982c91305d1bdd25533a52407, because 4.0.1 worked fine under the same conditions.

```
Xcode 7.0.1
Carthage 0.9.3
```

Cartfile:
```
github "Moya/Moya" ~> 4.0
```

```
*** Cloning Moya
*** Cloning Alamofire
*** Checking out Alamofire at "3.0.1"
*** Checking out Moya at "4.0.2"
*** xcodebuild output can be found in /var/folders/fp/2hwf2d3j6mjf8kz8h2v27zc80000gn/T/carthage-xcodebuild.APAymT.log
*** Building scheme "Alamofire iOS" in Alamofire.xcworkspace
*** Building scheme "ReactiveMoya" in Demo.xcworkspace
*** Building scheme "RxMoya" in Demo.xcworkspace
** BUILD FAILED **


The following build commands failed:
	CompileSwiftSources normal x86_64 com.apple.xcode.tools.swift.compiler
(1 failure)
<unknown>:0: error: no such file or directory: '/Users/Andi/Desktop/MoyaTest/Carthage/Checkouts/Moya/Demo/Pods/RxSwift/RxSwift/AnyObserver.swift'
<unknown>:0: error: no such file or directory: '/Users/Andi/Desktop/MoyaTest/Carthage/Checkouts/Moya/Demo/Pods/RxSwift/RxSwift/Schedulers/ConcurrentMainScheduler.swift'
<unknown>:0: error: no such file or directory: '/Users/Andi/Desktop/MoyaTest/Carthage/Checkouts/Moya/Demo/Pods/RxSwift/RxSwift/Schedulers/ImmediateScheduler.swift'
<unknown>:0: error: no such file or directory: '/Users/Andi/Desktop/MoyaTest/Carthage/Checkouts/Moya/Demo/Pods/RxSwift/RxSwift/ObservableConvertibleType.swift'
<unknown>:0: error: no such file or directory: '/Users/Andi/Desktop/MoyaTest/Carthage/Checkouts/Moya/Demo/Pods/RxSwift/RxSwift/Observables/Implementations/ShareReplay1.swift'
<unknown>:0: error: no such file or directory: '/Users/Andi/Desktop/MoyaTest/Carthage/Checkouts/Moya/Demo/Pods/RxSwift/RxSwift/Observables/Implementations/SkipUntil.swift'
<unknown>:0: error: no such file or directory: '/Users/Andi/Desktop/MoyaTest/Carthage/Checkouts/Moya/Demo/Pods/RxSwift/RxSwift/Observables/Implementations/SkipWhile.swift'
<unknown>:0: error: no such file or directory: '/Users/Andi/Desktop/MoyaTest/Carthage/Checkouts/Moya/Demo/Pods/RxSwift/RxSwift/Observables/Implementations/Using.swift'
A shell task failed with exit code 65:
** BUILD FAILED **


The following build commands failed:
	CompileSwiftSources normal x86_64 com.apple.xcode.tools.swift.compiler
(1 failure)

```

**Comment by justinmakaila** [comment]
Hey, can you post the contents of the `xcodebuild` output file?

**Comment by aloco** [comment]
This is fixed with 731db29d6a68d350ef97ad059e881f361dc35e3f, thx @bojan 
A version bump would be nice :)

**Comment by ashfurrow** [comment]
You just need a new tag for Carthage, right? Can do :)

**Comment by aloco** [comment]
Exactly, leaving my Cartfile with ```"Moya/Moya" "master"``` looks risky to me :)

**Comment by ashfurrow** [comment]
Cool, I've created https://github.com/Moya/Moya/releases/tag/4.0.2-carthage – let me know if that works!

**Comment by aloco** [comment]
thx! Although Carthage can´t pick this tag with ```github "Moya/Moya" ~> 4.0``` I think this is sufficient for the moment, since I can use `github "Moya/Moya" "4.0.2-carthage"` 

I hope there is soon a 4.0.3 release so I can use ` ~> 4.0 ` again :)

**Comment by ashfurrow** [comment]
As long as the new release didn't re-re-introduce the problem :sweat_smile: What did the fix turn out to be, so we can integrate it into our deploy script?

**Comment by aloco** [comment]
There were some files missed, but as already mentioned https://github.com/Moya/Moya/commit/731db29d6a68d350ef97ad059e881f361dc35e3f fixed that. So I guess there is no need for any deploy script update

**Comment by ashfurrow** [comment]
OK, cool. This commit ( https://github.com/Moya/Moya/commit/d0d30b1ce98972ce30d4a5b7b2be0aa3d7b43fe3 ) got released as 4.0.3 tag. My guess is an older version of CocoaPods was used to deploy the problematic release. Let me know if it works.

**Comment by aloco** [comment]
thx, everything seems to be working again 👍🏻
